web_action_conditionable/static/src/js/views.js

```diff
@@ -0,0 +1,24 @@
+odoo.define('web.web_action_conditionable', function (require) {
+    "use strict";
+
+    var View = require('web.View');
```

Comment:
I am not sure how this can work, I am not able to find any reference to this widget in `web/static/`. Also, I think this will need to be implemented differently, you can get an idea here: https://github.com/OCA/OCB/blob/13.0/addons/web/static/src/js/views/basic/basic_view.js#L246

Reply:
I agree that this is not the approach to follow IMO but I think the change has to be done somewhere related to the controllerParams variable: https://github.com/OCA/OCB/blob/13.0/addons/web/static/src/js/views/abstract_view.js#L136

What do you think?

Reply:
In JS debugging following error is raised:
warning: Some modules could not be started
boot.js:220 Missing dependencies:     ["web.View"]
boot.js:234 Non loaded modules:       ["web.web_action_conditionable"]

web.View does not exist anymore.